frontend/src/crosswordView/crosswordView.scss

```diff
@@ -0,0 +1,121 @@
+.crossword-title {
+    font-size: 50px;
+}
+
+.give-up {
+    margin: 10px 10px 10px 0;
+}
+
+.difficulty-button {
+    height: 100px;
+    width: 200px;
+    font-size: 40px;
+
+}
+
+// Clue Styles
+.clues-box {
+    background: rgba(105, 64, 238, 0.25);
+    border-radius: 10px;
+    padding: 0 10px 10px;
+    margin-bottom: 30px;
+    max-height: 300px;
+    overflow-y: scroll;
+}
+
+.clue-header {
+    margin: 10px 0;
+}
+
+.clue {
+    margin: 5px 0 5px 15px;
+}
+
+.clue-found {
+    text-decoration: line-through;
+    color: #a0a0a0;
+}
+
+// Definition Box
+.definitions {
+    background: rgba(105, 64, 238, 0.25);
+    border-radius: 10px;
+    padding-bottom: 20px;
+    max-height: 300px;
+    overflow-y: scroll;
+}
+
+.definition-title {
+
+}
+
+// Crossword Styles
+.crossword-grid {
+    display: flex;
+    flex-direction: column;
+}
+
+.crossword-row {
+    display: flex;
+    justify-content: center;
+}
+
+$cell-width: 50px;
+$cell-height: 50px;
+
+.crossword-cell {
+    background: #E8C2FF;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid #5941A9;
+    position: relative;
+}
+
+.correct-cell {
+    color: green;
+}
+
+.clue-num {
+    color: black;
+    position: absolute;
+    top: 1px;
+    left: 3px;
+    font-size: 16px;
+    font-weight: bold;
+    z-index: 3;
+}
+
+.blank-cell {
+    width: $cell-width;
+    height: $cell-height;
+    background: #2D2155;
+}
+
+// Cell Input
+.cell-input {
+    background: #E8C2FF;
+    height: $cell-height;
+    width: $cell-width;
+    font-size: 35px;
+    position: relative;
+    text-align: center;
+    font-weight: bold;
+    z-index: 1;
+
+    &:focus {
+        z-index: 2
+    }
+}
+
+.blank-input {
+    background: #2D2155;
+    height: $cell-height;
+    width: $cell-width;
+    position: relative;
+    border: none;
+    cursor: default;
+
+    &:focus {
+        outline: none;
+    }
+}
```

Comment:
This CSS looks well formed! There's a few topics that might be worth discussing, especially if any one has a particular interest in presentation. 

[Using relative units for sizing](https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/). This is a big aid in responsive design and worth thinking about, if you haven't already.

[Alphabetical ordering for CSS properties](https://medium.com/@jerrylowm/alphabetize-your-css-properties-for-crying-out-loud-780eb1852153). This is a big aid in avoiding headaches when you inevitably need to update old CSS.

Since I'm coming into this well into the project, please feel free to ignore if you all have common patterns that you're following already.